### ISSUE TITLE
Car routing should avoid recommending bicycle lanes

### ISSUE THREAD

**Issue by polarbearing** [osmandapp/Osmand#6807]
nightly OsmAnd-master-nb-2019-04-09.apk

In Berlin we have an increasing number of bike lanes in the middle of car lanes.
E.g. way 28698789 is tagged
```
access:lanes:forward = yes|yes|no|yes
bicycle:lanes:forward = no|no|designated|no
turn:lanes:forward = left|left|left|right
highway = secondary
```

That means the 3rd lane should be ignored for car routing. 
Currently the lane advice incorrectly prefers the bike lane.
(it correctly prefers the rightmost left lane, because a right turn follows)

![Screenshot_20190409-164424](https://user-images.githubusercontent.com/8266355/55892951-98ab0b80-5bb7-11e9-99ca-d703a193fb7c.png)

**Comment by scaidermern** [comment]
`access:lanes` is not supported yet, see #6074 for a similar issue.

**Comment by vshcherb** [comment]
Merged into #6074
